src/GraphQL/Types/Schema.cs

```diff
@@ -314,6 +327,8 @@ private IEnumerable<IGraphType> GetRootTypes()
 
         private SchemaTypes CreateSchemaTypes()
         {
+            FieldMiddleware.ApplyTo(this);
```

Comment:
Let's move `ApplyTo` into `SchemaTypes` to break recursive call to `Lazy.Value`.

Reply:
Also no need for `ApplyTo` to be `public`.